modules/ReactBreakpoints.js

```diff
@@ -49,8 +49,14 @@ class ReactBreakpoints extends React.Component {
   }
   state = {
     breakpoints: this.props.breakpoints || {},
-    screenWidth: this.props.guessedBreakpoint || this.props.defaultBreakpoint,
-    currentBreakpoint: null,
+    screenWidth: global.window
+      ? global.window.innerWidth
+      : this.props.defaultBreakpoint,
+    currentBreakpoint: global.window
```

Comment:
Please add a small comment above `currentBreakpoint: global.window ? ...` to describe what's going on here.

My goal with these comments is that even an inexperienced developer can read the code and figure out basically what is happening and why :)